src/transformers/models/seggpt/image_processing_seggpt.py

```diff
@@ -71,6 +109,10 @@ class SegGptImageProcessor(BaseImageProcessor):
         image_std (`float` or `List[float]`, *optional*, defaults to `IMAGENET_DEFAULT_STD`):
             Standard deviation to use if normalizing the image. This is a float or list of floats the length of the
             number of channels in the image. Can be overridden by the `image_std` parameter in the `preprocess` method.
+        num_classes (`int`, *optional*, defaults to `None`):
```

Comment:
Would perhaps call it `num_labels` (as we usually use that for number of classes in the Transformers library)

Reply:
changed here https://github.com/huggingface/transformers/pull/27735/commits/0c9ad32791eab41505d75589ebc091679a7646e7